### DISCUSSION TITLE
Certain keys seem to be missing from the Copy-Mode key table

### DISCUSSION THREAD

**Discussion by FluxZA** [wezterm/wezterm#2998] [Q&A] [answered]
Hi

I was looking into copy mode and trying out the key bindings when I realised that `e`, `ctrl-u` and `ctrl-d` are mentioned in the documented defaults, but they don't work in copy mode. When I use `show-keys` to view the current key table, they are not their, and they are also missing from the example key table in the docs. Therefore, I can't rebind them myself, as I don't know what the correct commands are.

Have I missed something, or are the docs slightly out of line with the current functionality? (I am on the stable version for Fedora 36)

**Comment by wez** [comment, marked as answer]
Those are new in the nightly build, so it sounds like the version info is missing from the docs. 56dad40f7c62ddb0f6fb2d0287174611409e713f fixes the docs

**Reply by FluxZA** [reply]
Thanks for the fast reply and for clarifying.
